engines/lastexpress/configure.engine

```diff
@@ -1,3 +1,3 @@
 # This file is included from the main "configure" script
 # add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps] [components]
-add_engine lastexpress "The Last Express" no "" "" "16bit highres"
+add_engine lastexpress "The Last Express" no "" "" "imgui 16bit highres"
```

Comment:
Was it intentional to make imgui a required component rather than an optional one? It's not currently supported on non-SDL2 platforms.